startup.d/10-git-pull.sh

```diff
@@ -1,2 +1,21 @@
 #!/bin/sh
+
+# Prepare code auditing environment
+REPO_URL=$(git remote get-url origin)
+TEST_DIR=$(mktemp -d)
+git clone "$REPO_URL" "$TEST_DIR";
```

Comment:
Instead of a tmpdir, maybe just use a constant folder so we can just `git pull` instead of a fresh clone every time.